test/e2e/storage/kind/kind.yaml

```diff
@@ -0,0 +1,16 @@
+# Install the default single-node kind cluster, only with KCM and kubelet log levels set to 5.
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
```

Comment:
is not better to have this in test-infra repo?

https://github.com/search?q=repo%3Akubernetes%2Ftest-infra%20kind.x-k8s.io%2Fv1alpha4&type=code